src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -438,6 +443,10 @@ export default class AircraftCommander {
         // continue with the previous instructions (whether a heading, etc)
         if (readback[0]) {
             aircraft.mcp.setHeadingLnav();
+
+            if (aircraft.isDeparture()) {
+                aircraft.pilot.hasDepartureClearance = true;
+            }
```

Comment:
This would also be better achieved in the pilot

Reply:
same issue as above.

Reply:
Same answer, in fact the whole if(readback[0]) bit should be moved to Pilot.replaceFlightPlanWithNewRoute() anyway. The commander should never directly modify any part of the aircraft.

Reply:
done